typespec-tests/src/main/java/com/cadl/armstreamstyleserialization/fluent/models/SalmonInner.java

```diff
@@ -230,14 +254,35 @@ public String pattenAnotherPropertiesPatten() {
         return this.innerAnotherProperties() == null ? null : this.innerAnotherProperties().patten();
     }
 
+    /**
+     * Get the requiredString property: The requiredString property.
+     * 
+     * @return the requiredString value.
+     */
+    public String requiredStringAnotherPropertiesRequiredString() {
+        return this.innerAnotherProperties() == null ? null : this.innerAnotherProperties().requiredString();
+    }
+
+    /**
+     * Set the requiredString property: The requiredString property.
+     * 
+     * @param requiredString the requiredString value to set.
+     * @return the SalmonInner object itself.
+     */
+    public SalmonInner withRequiredStringAnotherPropertiesRequiredString(String requiredString) {
+        if (this.innerAnotherProperties() == null) {
+            this.innerAnotherProperties = new AnotherFishProperties();
+        }
+        this.innerAnotherProperties().withRequiredString(requiredString);
+        return this;
+    }
+
     /**
      * Validates the instance.
      * 
      * @throws IllegalArgumentException thrown if the instance is not valid.
      */
-    @Override
```

Comment:
Any reason we remove the `@Override`?

Reply:
Right, it was not intended.. This is controlled by a flag which also controls whether to call `super.validate()`. 

Let me add it back.